build/purser-install.sh

```diff
@@ -0,0 +1,86 @@
+# Realease Version
```

Comment:
Update installation and uninstallation steps in Readme file. Also mention prerequisite binaries for the same(kubectl, wget, etc.)

Reply:
Updated and created a new PR for README changes #14